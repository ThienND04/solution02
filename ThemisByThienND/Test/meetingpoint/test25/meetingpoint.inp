994 1632 12
527 582 215613
736 187 47791
472 313 270094
177 661 211148
717 498 465188
495 574 169564
437 301 503222
240 645 4334
586 846 134001
102 790 124408
608 253 45351
181 45 72630
324 612 61692
967 604 37180
376 893 451598
104 727 65288
339 850 417683
898 134 542651
56 229 16598
220 29 391254
389 721 126609
743 610 95634
705 806 44212
532 862 102151
718 50 410766
314 658 71017
313 274 104820
883 517 41162
474 78 227582
896 714 317837
566 626 307478
968 294 356055
822 568 36168
799 611 88227
712 636 258108
664 60 32647
387 609 490245
729 492 154551
299 47 11051
403 612 114979
898 892 606187
446 838 58091
782 358 267422
447 856 187942
78 175 216403
814 395 132498
758 766 424558
463 362 138856
373 375 82815
530 977 589218
925 102 293287
211 321 144074
963 840 367713
233 23 157133
44 32 332528
169 288 68888
745 208 2109
777 656 244117
209 106 226853
784 320 296953
438 574 16981
654 878 288210
720 587 85624
435 569 195771
643 156 9583
205 762 342385
727 421 304931
858 368 343316
625 82 606553
43 934 9329
264 285 17402
42 29 34426
340 778 28832
837 943 245160
824 539 14337
156 494 37971
783 955 12183
546 374 331090
193 912 540049
831 227 734635
466 373 440430
742 335 446024
916 918 45877
559 379 130987
808 979 145844
906 391 126942
865 730 147094
735 236 111490
835 986 791663
677 210 287937
261 902 1000000
45 616 589208
739 119 143165
638 430 134081
948 312 637737
315 664 669401
725 35 156261
450 591 36991
914 718 395682
357 354 131427
695 234 369118
203 763 355027
875 708 165817
305 526 103334
907 37 30127
392 261 513052
602 342 24031
829 814 45303
542 651 43790
176 800 132845
181 396 368109
789 341 288506
372 670 6783
482 701 408868
670 701 557778
109 876 136728
877 691 187126
78 51 92537
820 408 73603
305 443 422685
792 666 548764
544 863 128410
64 803 185894
9 157 168409
861 409 244841
70 80 6142
217 401 517492
138 457 224618
371 916 34279
31 976 172756
468 730 181518
83 735 115064
312 918 2758
427 979 312320
771 306 428324
696 41 74979
228 686 463568
622 418 84636
399 128 65618
826 525 54808
915 770 100357
665 933 25623
969 427 1023
111 527 371238
445 460 17738
976 766 8791
413 597 63738
942 600 1782
855 529 416417
150 739 190497
395 857 8512
585 547 91805
890 151 107750
108 321 24278
52 134 111426
48 433 111267
659 411 31057
617 763 397349
6 269 574043
918 395 33384
859 371 45499
415 828 214668
969 372 119822
554 61 511167
789 835 668481
994 502 765931
255 909 233095
883 138 590455
328 796 18887
254 682 241120
348 519 3864
175 955 31152
454 6 9074
511 418 60296
812 899 387758
568 316 28861
162 286 89904
279 881 307520
838 612 216224
962 787 358492
625 842 251499
949 614 535829
585 372 151071
331 22 260682
81 984 111667
125 281 561113
984 778 493284
456 189 123445
963 398 64602
566 188 174894
347 526 127418
68 674 205747
571 163 178474
415 275 208183
581 612 384339
926 385 4289
973 936 44984
138 600 32524
538 36 92749
529 171 11599
539 618 696
768 332 41000
364 71 265410
253 582 419122
438 428 285359
952 617 47473
178 502 12953
544 445 95492
201 889 103553
581 164 391359
846 830 52222
229 88 33990
544 516 229873
156 24 538131
231 356 7563
99 905 28764
430 922 281518
230 405 184781
474 629 74681
700 345 328624
952 606 273739
117 151 301225
159 847 26184
694 807 113898
720 220 323865
570 934 236727
193 630 584206
912 254 569223
863 707 98059
708 522 47328
385 986 475523
615 314 24502
496 17 295056
97 349 142767
537 263 189446
625 632 108592
57 77 155222
675 461 199441
233 403 54872
213 327 216679
641 751 533603
749 421 457299
768 820 446815
918 22 14669
282 224 460431
108 474 198227
939 98 110353
641 555 284690
83 876 63813
105 870 77593
806 715 403813
813 969 143768
918 617 180719
729 741 78037
557 370 254270
108 548 35911
25 369 8173
798 842 240588
580 310 120079
795 730 746423
458 150 198505
847 54 337317
342 763 385974
830 986 544147
211 745 328867
215 117 191290
618 32 109395
47 138 469493
55 19 403717
382 983 427851
144 657 230421
387 91 17887
690 532 294909
729 405 71673
916 374 17131
310 252 159376
879 161 449811
394 934 49049
297 449 322853
692 470 40033
60 137 37759
501 465 42374
13 720 227540
884 319 422286
64 404 327176
505 12 134214
664 697 184887
984 726 456674
879 874 229220
427 121 35320
869 755 287442
263 749 18786
425 131 8646
242 844 642354
713 143 508480
271 741 197815
138 618 38174
549 349 28230
735 650 10678
873 378 46150
153 5 315405
176 740 277382
35 736 183647
719 325 501212
765 173 370220
415 777 150146
79 983 386378
429 319 39325
941 987 108856
522 983 171073
472 227 543876
497 179 40076
787 979 6465
203 776 389026
510 440 11270
982 698 4967
973 172 24236
148 56 237423
497 521 244216
663 610 514962
384 435 39129
736 319 55284
971 889 386279
773 976 582825
55 335 1818
986 581 210991
112 495 150792
487 781 135091
431 524 173187
951 170 480687
92 729 46191
77 226 63883
537 786 279420
264 677 389810
802 862 252142
811 538 229238
79 338 437113
367 454 97852
685 764 96761
665 428 31697
258 423 607931
993 619 145964
73 367 81470
792 384 16877
55 257 109682
722 118 337731
835 631 125563
828 637 321583
804 617 398325
374 539 30387
815 701 623036
596 34 84512
122 515 189016
886 907 134137
207 648 97299
956 423 441000
640 455 263488
838 112 4954
403 623 353632
803 247 49368
649 522 131253
612 718 4056
384 1 121116
738 699 66114
240 943 82876
162 584 51823
969 207 94745
740 450 110481
588 166 368106
749 953 384066
259 420 142978
292 771 12048
354 711 104669
494 15 496546
622 783 458702
482 288 114659
828 174 12526
322 280 516640
806 837 269316
301 187 44596
59 843 391044
245 614 1437
66 945 24863
988 88 168753
937 544 94579
70 80 6142
359 226 181424
816 344 304743
866 176 152427
283 823 133861
865 222 291875
313 425 117412
587 528 83761
529 759 410186
261 174 535524
159 288 44619
605 157 12962
939 649 54955
921 218 200421
514 887 107522
935 656 204630
860 454 594543
512 568 86993
67 56 74181
94 404 348721
726 828 39610
653 880 174855
735 868 601561
578 190 340087
59 258 631142
467 105 312198
54 432 91427
109 719 573050
705 550 553505
199 969 23115
527 713 3353
641 270 285787
949 513 318997
113 485 559001
399 22 19276
664 593 34583
647 830 330428
717 772 480590
712 691 94315
937 665 9951
225 455 66445
97 280 52063
246 518 117152
530 253 41916
747 367 479827
224 872 498562
15 134 527402
840 567 549557
187 246 74025
256 795 127305
685 260 76348
735 134 75175
783 406 15173
864 200 57035
56 113 29668
316 671 285232
710 549 440646
137 823 58063
469 599 140657
675 198 252430
169 199 68851
75 970 106763
375 150 91756
45 708 9830
760 727 173691
171 685 92823
835 258 120220
58 179 315543
799 660 4588
444 226 254839
253 141 6646
121 333 85478
517 56 635887
319 478 402921
764 590 584143
21 463 173090
536 507 329057
400 39 501391
207 557 360699
464 485 389523
384 165 441249
474 567 13489
561 927 315425
775 598 215628
230 109 66029
682 226 230593
990 892 23959
355 392 364840
377 336 61724
835 918 100311
509 323 43121
291 820 31284
311 727 13972
7 231 116621
802 929 197850
730 23 99423
455 701 640824
694 755 57222
336 73 356738
255 322 165782
657 917 43621
23 136 72662
825 72 297925
855 769 19062
405 342 117108
771 193 25434
811 495 425476
388 318 393480
315 843 410851
103 280 521657
739 378 148253
672 640 37150
407 342 373540
83 833 127075
616 541 11309
175 109 9989
157 426 45702
588 223 79290
64 292 166086
891 704 163839
500 281 159610
180 641 141345
742 717 63895
594 492 417582
643 974 55892
127 740 122592
262 258 330539
705 676 430990
566 223 2508
125 493 99188
239 556 178909
367 301 31750
989 544 561460
492 276 583544
169 889 207158
270 652 266252
92 328 18342
250 187 209419
705 926 188687
174 564 243258
764 708 115081
981 160 581744
793 913 194733
577 150 35395
51 734 111017
66 491 242350
542 406 144721
421 397 303893
620 325 151500
210 662 26900
200 353 53108
708 538 336901
737 380 91870
299 193 412872
484 382 10327
28 490 209084
670 86 185239
408 766 209102
477 979 155326
507 174 313982
917 194 170417
583 504 157822
887 843 101588
844 285 51665
145 28 369930
156 618 86643
684 972 485589
325 601 2885
826 930 363821
374 916 17131
463 438 127985
115 786 446805
818 751 318981
146 808 42666
489 333 208744
91 255 157428
620 115 144394
328 963 59562
347 22 329400
341 283 131999
868 124 455948
497 774 163462
774 691 610909
551 950 437169
680 518 26570
711 752 365513
817 202 132399
43 507 247264
480 33 215333
249 834 209052
586 138 297594
374 273 1555
775 126 356608
879 621 435952
325 851 477981
601 213 153049
531 373 55631
306 946 122985
307 313 44760
660 697 160646
189 538 263246
791 935 235774
808 81 118675
168 432 59082
180 141 122356
362 922 37595
685 991 7358
870 426 44844
702 927 290871
738 846 299786
170 751 354687
446 860 420022
240 785 133206
450 47 450199
122 795 541988
370 186 478442
927 152 74704
336 938 45260
779 924 377563
397 489 39321
214 68 246287
211 155 13168
271 620 231653
429 234 140241
263 722 1472
894 661 13752
61 688 206344
295 184 727965
203 644 280367
722 434 172600
471 63 447939
834 361 173214
329 274 152988
543 952 461980
180 58 424917
226 371 4703
872 896 12530
830 48 11740
722 366 127854
95 142 295456
520 133 65598
320 872 442471
530 361 31751
728 255 188338
553 278 520637
744 739 41062
239 553 30914
958 267 278626
147 939 32149
273 498 70456
632 702 323184
704 2 199194
430 133 505829
225 797 409358
163 393 27813
585 755 264030
937 875 145714
504 507 217270
985 874 156078
23 709 50614
683 866 94290
768 109 580626
478 241 428250
506 492 178918
741 563 13586
611 889 188061
460 427 20037
591 923 590702
646 611 345158
845 227 309390
378 340 325412
393 456 49015
322 536 20486
769 24 108221
956 402 112750
192 797 39299
729 825 265173
636 704 141763
965 200 245624
981 565 390624
568 369 59461
266 320 394290
56 298 294268
265 519 755254
977 30 638173
764 853 111920
1 835 809
313 141 126998
118 836 10801
115 632 11092
87 488 459739
261 918 50782
629 577 163878
557 575 213747
361 332 470632
904 897 174057
729 492 154551
774 253 21543
98 345 179219
411 592 101496
767 836 217949
878 530 116030
136 422 211154
217 925 110086
33 39 8169
44 840 160686
479 222 171413
924 586 31078
770 793 385330
961 379 33176
591 742 409929
910 453 114326
57 453 12500
514 338 132607
8 701 425068
784 258 454599
784 853 461835
729 215 47549
293 592 164
235 596 41643
56 735 15322
633 910 397422
275 705 119499
370 41 390978
781 909 496247
581 656 27278
123 679 13348
740 904 472279
446 823 240354
26 743 141664
934 950 231393
864 272 198079
993 395 222916
446 359 12129
280 421 486137
539 925 4129
299 900 171440
345 747 339076
592 872 283546
32 513 241460
560 149 34346
475 778 380291
883 690 30089
69 597 74348
87 34 56126
737 385 95072
761 411 28775
303 551 96002
72 908 188447
772 463 385673
764 369 305081
394 490 213175
987 37 378455
247 323 44502
911 600 288068
743 186 634175
137 456 141071
45 436 54278
609 250 205179
748 673 335172
722 373 19857
990 466 525194
364 993 52623
978 113 199962
309 18 112612
625 135 53414
171 519 204729
669 87 209468
56 189 57923
759 567 438212
34 93 37113
903 545 254141
992 706 2385
226 906 550779
162 455 474344
534 628 30521
755 835 105443
661 36 3267
935 648 58134
429 485 376694
990 600 16515
335 173 85044
235 984 51310
243 576 93053
455 622 446303
912 793 359239
976 664 46456
267 1 120334
498 674 465894
662 805 244578
691 29 629370
472 75 167001
839 849 366091
114 207 201664
185 813 209923
328 177 58074
853 474 24549
573 885 67408
865 302 119753
687 505 360645
241 201 14959
663 816 24476
388 967 231955
428 587 19118
553 468 70678
844 862 195364
183 397 61701
83 439 522443
771 38 8768
162 509 293353
260 350 94973
698 746 384126
334 741 355937
725 355 289842
111 850 147359
183 959 126402
395 631 8132
264 385 142986
346 338 413603
450 835 155439
304 822 57509
151 692 475404
57 743 260249
814 116 121658
664 724 464192
827 784 119770
93 342 73112
990 696 463600
108 147 18227
961 120 362279
56 166 575852
156 764 183550
514 455 9062
908 798 493406
371 785 560295
653 574 124239
145 371 36952
774 130 441418
264 196 394002
583 968 180745
432 111 341460
730 878 312092
469 609 385217
526 654 28387
473 26 262009
24 433 375703
598 539 473013
113 230 63187
8 223 97566
447 431 62675
416 163 194795
757 880 22735
162 543 173995
148 395 235606
991 553 146894
944 38 288762
62 324 315581
142 989 358379
974 157 71599
373 163 153156
912 418 70783
521 496 141139
929 441 182900
907 681 104618
193 279 214133
518 333 26216
247 232 537880
182 541 142483
314 764 152167
136 839 18250
492 428 25293
797 776 94431
873 712 284588
368 529 189333
425 227 156368
147 357 112570
152 223 76169
209 970 50835
626 546 110175
290 564 12003
768 769 124803
825 420 107437
877 266 72802
330 494 61612
491 164 24797
490 581 202407
187 603 113926
511 698 159429
952 641 200111
930 333 536925
4 56 70775
156 408 181505
783 371 50721
441 737 50223
4 211 13314
859 113 64421
730 844 104203
499 216 359351
562 436 161180
748 950 59307
681 77 365983
155 471 376105
300 77 430366
233 143 20490
625 803 68181
933 969 2091
664 681 513270
867 905 550341
932 416 561731
307 714 95889
587 651 194554
89 861 232016
218 841 157885
627 532 360198
198 464 20761
226 676 377561
853 829 107906
8 220 190469
227 732 118312
427 520 7416
855 797 22480
381 234 546878
787 234 212734
204 452 27176
853 921 227531
694 378 291064
657 584 157312
748 929 52997
23 831 635928
375 663 434888
194 135 109179
562 240 266119
341 497 40938
458 445 474661
897 913 374249
840 819 73214
411 419 46750
804 17 57810
685 303 590777
417 581 106902
967 558 357842
61 274 37398
151 301 29409
661 11 102734
412 652 130932
661 292 376194
673 33 581371
317 767 62548
153 192 199417
666 982 107036
259 618 469501
794 755 505597
90 769 267211
510 727 46994
714 809 265690
908 901 592014
967 442 55856
942 77 78602
867 768 22344
80 765 466451
599 893 49509
727 209 167135
396 432 192269
807 476 94219
129 715 190803
527 121 655883
921 368 27866
35 937 296638
383 370 129215
547 856 146738
293 915 232974
888 449 103791
486 610 470228
177 98 92051
52 243 31956
245 655 15301
494 936 515544
277 452 156135
719 265 418878
879 928 514811
580 45 41771
694 862 136784
550 488 12798
567 160 35581
670 463 346149
756 199 487433
739 943 88446
63 815 2629
833 437 390574
299 457 255926
455 34 40036
30 918 46678
445 484 616736
937 491 28265
34 6 6838
978 241 82310
671 112 153171
64 730 366
705 169 58809
65 362 17119
983 46 225313
33 118 394247
275 189 61440
966 735 554062
680 198 39113
451 375 134915
454 428 21949
894 858 155419
274 10 427672
227 210 274428
273 619 80037
993 527 432151
529 840 521531
228 577 384155
686 859 70715
226 913 4724
326 908 467425
746 772 4157
661 906 193420
711 583 28735
915 881 55870
800 759 86915
733 30 451240
302 758 314844
63 816 572846
932 785 16249
467 39 388441
53 396 7181
936 446 383665
914 689 197284
51 418 400504
648 624 166374
763 574 252006
607 48 8550
731 680 169891
126 11 9094
39 788 33665
274 422 213564
925 170 204511
676 621 216730
953 173 377063
878 838 100581
14 109 29020
618 1 98780
113 41 529188
448 85 48501
587 641 10483
7 859 78196
642 462 46667
539 259 468806
864 543 382001
47 895 360529
859 932 531045
420 844 606365
614 169 41447
690 616 18672
248 727 284640
337 631 163303
781 969 194657
88 212 65893
155 221 9057
312 172 565840
191 871 185060
368 256 250617
741 363 158967
668 258 15563
480 135 204501
266 91 41666
646 82 169709
540 849 63028
666 711 282603
805 489 119404
397 334 384035
738 229 484247
452 908 294535
110 833 121429
42 618 24714
710 497 426085
132 760 160749
909 121 656607
214 882 160175
503 712 51567
331 873 31916
392 62 90541
579 788 569268
664 1 102576
645 964 172618
885 755 91954
711 61 196387
730 410 256087
219 963 211806
545 852 307362
613 388 23578
202 671 301325
348 718 198854
234 959 119469
124 658 37917
870 403 11239
450 60 21024
398 54 270088
64 854 142808
198 410 54291
664 38 67847
237 59 25359
394 442 187057
511 201 439221
548 441 200830
576 141 280783
739 510 262236
4 750 380040
743 33 16519
957 647 194540
644 951 184275
834 255 155484
175 364 155274
554 280 512589
748 720 182239
893 315 9905
388 780 486980
266 312 569086
1 730 691
300 907 40235
572 780 90113
858 733 113470
521 587 444373
715 402 179865
428 612 38336
558 906 29895
712 183 242685
755 625 12517
419 450 133105
771 820 114130
839 352 107215
719 55 577236
518 231 176208
110 167 300963
319 878 7449
917 348 51729
123 206 94923
215 9 232108
605 848 103652
16 88 265499
888 693 32436
183 218 91455
39 610 70946
616 263 499596
577 887 117365
743 395 10329
176 156 358689
892 42 5508
9 223 107051
730 703 789829
461 107 53884
451 695 118079
97 427 55914
10 383 426691
204 929 30256
258 613 32249
954 942 171662
836 106 216965
698 574 147574
835 153 379231
991 705 26556
418 442 320937
903 199 19627
694 545 232803
908 9 36640
459 595 308374
866 559 280722
376 445 194139
471 645 2020
198 861 26443
990 635 91289
985 260 534637
753 534 301333
822 282 471694
507 49 355578
184 530 225708
172 811 13377
722 413 159970
169 482 183548
935 71 220636
802 597 110826
261 244 164510
974 254 75446
893 51 508495
980 448 228354
232 993 416779
539 734 62893
994 536 450345
425 191 6519
521 230 358518
790 494 49090
480 142 58031
174 373 449698
251 543 489678
703 926 469282
33 168 539359
106 457 33739
352 783 167690
218 424 182682
225 672 159892
835 141 147382
448 495 93995
820 897 271629
666 366 341035
506 513 86620
30 365 464085
889 763 280670
818 82 381500
897 892 435948
696 480 239876
399 493 467744
628 788 82517
460 958 280556
748 138 270595
564 497 104324
567 828 523066
662 166 405803
207 416 1950
561 617 293262
64 839 189970
254 569 180565
895 56 113232
686 607 69456
498 683 400374
803 152 1980
169 909 598937
86 432 348557
29 383 13429
254 312 71573
638 754 248521
169 902 960543
772 492 2376
470 366 365911
932 683 155569
905 589 267976
748 466 270607
110 152 53102
159 49 424154
620 3 301033
609 806 29735
23 185 352970
334 230 547083
618 448 64843
970 161 272571
426 919 241972
159 40 265476
400 894 113825
486 722 492830
554 183 268644
122 394 39227
680 44 268748
320 877 321488
381 994 401549
168 304 431880
475 137 197846
80 149 8899
233 826 135024
369 12 53533
364 387 391947
238 643 35320
736 249 360722
690 447 276435
907 180 449916
978 288 102068
114 826 3528
931 725 167223
540 263 403751
35 716 151837
459 190 502803
623 554 53183
483 846 74933
435 852 543393
100 227 189970
149 785 586712
18 625 23026
126 489 3598
729 462 250740
353 342 5932
684 529 99466
308 841 466507
934 291 145189
358 229 222881
721 626 17866
208 589 112323
868 559 303106
448 533 40784
556 554 256223
526 916 425613
144 251 123877
505 451 67825
165 937 425308
171 49 537563
849 338 272497
173 639 425884
36 695 26931
954 821 339611
441 708 54696
699 201 371484
390 881 59684
144 491 12415
257 636 169859
379 823 169910
185 268 235048
199 236 106345
865 607 80248
629 715 367551
745 218 3896
115 366 107268
954 583 89968
934 197 89592
709 960 306364
318 320 291662
76 258 22455
694 63 5365
302 406 121810
402 102 3817
536 217 488810
813 567 91327
654 229 484908
659 932 390658
631 724 440396
806 226 9215
730 325 230658
886 192 6869
588 37 410159
940 258 10517
562 774 165863
141 263 68205
829 956 66259
486 14 522164
352 728 70621
329 595 74591
457 806 184809
823 664 12343
320 38 108251
678 734 402368
156 988 133407
600 747 588954
397 78 22663
107 916 163181
146 416 111422
720 414 31377
503 853 630751
857 656 411184
807 574 149807
934 23 16733
552 300 298108
520 706 58173
948 340 38336
640 411 70337
600 718 31719
36 657 138445
763 688 265077
388 62 397558
894 648 265651
870 320 44682
223 850 9833
386 550 360062
926 139 416415
161 87 93835
969 281 41494
637 803 166363
755 348 293799
491 84 104060
880 819 48326
387 572 57206
314 182 408159
350 756 339984
325 947 528748
317 229 148215
30 931 609383
390 330 500954
832 190 502309
89 799 16731
135 138 35230
921 33 179972
951 162 171507
3 201 388918
847 200 21203
720 924 178160
324 941 4446
81 258 25140
409 557 27518
414 818 87136
353 69 68084
479 129 160305
226 106 141854
497 970 265679
328 253 119278
156 523 401598
99 199 16180
638 210 267294
662 189 112125
645 224 381444
215 508 57874
899 823 86614
364 295 845630
427 268 119666
224 615 44041
31 66 85884
599 826 340101
368 967 39621
449 261 284448
234 226 18439
137 134 19555
653 456 237367
467 796 264959
522 457 197288
491 360 216423
947 321 399248
164 293 247314
600 101 408246
641 568 118288
982 248 92467
440 557 203009
212 168 484742
871 881 376923
703 340 10566
447 499 310247
501 229 82640
158 386 74994
600 452 242505
285 240 451802
74 578 146810
613 903 11143
397 728 19724
170 535 96132
373 433 110124
178 157 51461
381 300 71067
552 592 55479
14 140 525479
653 234 347003
294 171 66124
323 721 308172
928 221 47064
926 496 121681
692 650 644984
711 243 68234
794 355 311734
533 316 71944
51 380 40357
180 9 116457
170 703 482405
650 172 563035
205 249 88411
22 344 310422
546 223 199811
96 650 231536
343 702 143770
232 978 411884
922 372 176480
974 937 113842
460 455 24546
20 900 221948
842 760 190973
841 747 58800
171 510 149098
140 912 30664
63 766 285
501 953 420522
744 651 227492
76 473 405023
395 277 121344
484 458 142079
788 605 106348
603 243 19736
481 851 55596
641 384 8084
955 991 12376
549 565 53253
959 667 235720
707 724 108660
77 181 50447
605 855 286739
305 910 372686
149 913 35845
694 130 454323
121 70 6126
430 115 438757
321 140 353163
908 991 11455
355 134 107576
970 90 257397
892 252 319008
504 977 195258
723 534 429505
295 241 870422
88 466 528079
713 931 29039
346 934 581378
331 53 183870
733 275 346988
813 704 333472
554 675 155434
256 182 57608
874 96 215321
126 275 179459
525 120 233204
98 207 46000
885 354 101278
284 445 211441
42 464 208336
12 180 17128
443 110 15184
332 698 124380
676 330 278533
539 621 247334
910 156 117833
766 552 224296
103 233 470755
285 99 17872
249 524 425749
568 940 115944
770 693 198187
7 762 51827
799 385 194719
232 673 31478
875 37 467819
238 877 369835
227 427 96960
539 229 16610
510 194 59192
308 619 21670
261 585 218380
896 276 33965
320 733 320363
650 918 5563
455 214 72781
223 764 259268
691 24 63735
20 455 81156
152 468 2762
606 211 386277
645 387 92716
187 763 476930
975 107 433274
662 113 140380
297 813 516045
478 810 288057
524 650 78338
97 822 57039
982 493 23086
899 625 82764
399 581 446413
548 19 186561
437 203 425915
279 965 179120
952 696 287250
516 709 310780
777 648 18647
251 194 20249
567 525 295248
360 27 19198
796 63 134892
19 393 195975
164 365 421763
377 103 50535
518 806 68622
338 203 39539
2 24 52271
246 229 12146
909 631 663924
372 335 62070
782 886 20061
296 552 12500
154 383 544194
847 801 281646
74 358 351451
448 758 418760
431 622 231046
227 835 599
873 623 257362
679 416 47053
216 122 166655
990 276 614679
399 134 57471
156 461 178638
222 35 5963
634 934 97845
757 879 33749
202 971 524540
147 708 201388
945 554 286398
153 333 160871
612 412 353167
543 708 674338
655 395 115041
889 128 197831
386 899 124459
90 25 32999
195 89 99828
261 212 64067
55 668 18733
280 10 391695
934 289 91873
70 795 607298
920 155 477430
199 73 237568
990 337 136712
241 670 54007
593 12 116258
43 887 133359
809 587 541846
708 529 62903
447 525 26196
2 865 628042
534 771 75991
238 765 382081
732 287 606897
684 860 495971
544 505 155995
432 772 573321
58 920 11912
343 373 169049
89 23 5382
853 19 445250
513 76 306152
138 274 33415
93 857 77085
391 311 208291
236 459 477471
336 164 526693
336 631 590441
764 730 1181
221 164 30764
810 411 84094
899 66 206139
358 308 105840
225 105 36365
131 670 59575
877 567 448330
122 981 564033
423 375 408417
873 993 50968
925 351 193923
388 836 367942
346 865 550440
399 190 68453
46 759 29374
471 372 386835
202 610 19788
918 642 441602
732 535 284528
421 955 574113
175195 826325
333811 666698
232911 58260
214240 714110
616843 433810
446269 893253
621738 223360
46932 17907
164767 227180
695265 622460
646191 87281
549365 54851
