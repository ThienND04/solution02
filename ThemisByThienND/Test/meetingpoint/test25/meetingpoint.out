201971628605
267711915202
54715170300
246984227360
332268527110
358684848897
192449050092
15101098449
112078303041
396266127980
88658817866
57536669513
